### ISSUE TITLE
Windows 3.0 multimedia does not start with any Sound Blaster Card enabled

### ISSUE THREAD

**Issue by Fenix770** [86Box/86Box#952]
**Describe the bug**

Windows 3.0 multimedia do not start with any Sound Blaster Card enabled, instead:
It returns to the DOS prompt if it's in 386 enhanced mode. Or locks up the emulated machine in the real and standard mode.
But works fine if the Sound card option in 86box is set to 'none'.

It occurs on latest commit of 86box regular branch, and also tested 86box v 2.07 and PCEM v16.
Works in PCEMv16 on Packarbell 486, only if the standard mode (WIN /S) is specified at the DOS prompt.
I also tested the standard VGA drivers but with no avail.

Here is the image file and a nvr and 86box:

[86ox-cfg-nvr.zip](https://github.com/86Box/86Box/files/4991501/86ox-cfg-nvr.zip)
[Win 30 dos 40 type 35.zip](https://github.com/86Box/86Box/files/4991503/Win.30.dos.40.type.35.zip)

**To Reproduce**

1. Configure 86box the video card as Video 7 1024i with 512k ( Win 3.0 is confured with V7 512kb driver)
2. Start the machine typing 'WIN' at DOS prompt.
3. The emulated Sound Blaster should be Sound Blaster Pro v1 to be compatible with the Win drivers.

**Desktop (please complete the following information):**
 - OS: Win 10 1909
 - 86Box version: Tested latest commit, (also 86box v 2.07 and PCEM v16)
 - Build type: Regular

**Comment by driver1998** [comment]
If you are talking about the Tandy version, the Sound Blaster is preset to a wrong config. And when FINDCD.EXE (Tandy's custom shell and demo software) starts, it tries to play a wave file, which crashes the system.
Change the shell to PROGMAN.EXE, set up the sound driver correctly in Control Panel, then change the shell back to FINDCD.EXE if you like.

**Comment by Fenix770** [comment]
This image is old and it worked before, but it was created more than a year ago, so I don't know if something changed in the emulation or in the configuration of  Windows.

**Comment by Fenix770** [comment]
Strangely the hd I uploaded don't say multimedia on win 3.0 startup screen but has multimedia aplications.
The Tandy versions work but crash when I exit to dos.
I installed a generic multimedia version, (no Tandy)- It allows to specify the SB card type at startup (SB 1.0/1.5/2.0/Pro) instead of the 'Tandy sound 16-bit option". And it works perfectly. Maybe the Tandy version requires a non-standard SB card.
